platform/src/ToolsManager.js

```diff
@@ -50,6 +48,14 @@ class ToolManager {
                         toolUrl.url = url;
                     }
                 }
+                else if (this.isValidUrl(url)){
```

Comment:
Shouldn't this case also capture `{{ID-...}}` cases?

Reply:
A new case is added to handle {{ID-...}} placeholders.